lib/ui/painting.dart

```diff
@@ -1740,6 +1740,17 @@ class Image {
     return Image._(_image);
   }
 
+  /// Returns true if `other` shares the same underlying image memory as this,
+  /// even if this or `other` is [dispose]d.
```

Comment:
might be worth referring to `clone` in these docs

Reply:
Done